src/redefined.resolver.ts

```diff
@@ -57,6 +58,22 @@ export class RedefinedResolver {
             this.resolvers.map(async resolver => {
                 try {
                     const accounts = await resolver.resolve(domain, networks, options);
+
+                    if(resolver.vendor == "redefined-username" || resolver.vendor == "redefined-email") {
```

Comment:
i think we can register "example.sol" only with redefined-username. Therefore, the redefined-email is not needed.

Reply:
it's processing bindings
ETH - 0x12321
SOL - 888.sol

Reply:
ok